fmn/config.py

```diff
@@ -0,0 +1,6 @@
+"""This module is responsible for loading the application configuration."""
+import fedmsg
+
+
+#: The application configuration dictionary.
+app_conf = fedmsg.config.load_config()
```

Comment:
I recommend against having the import of this module cause config loading to happen. It makes testing tricky. IIRC, Celery has events you can use to load the config during process start up.

Reply:
Totally agree. I started writing a lazy-loaded config, but that would have spiraled out of control. Before, every module was loading the config this way separately. I figured this would be a nicer middle ground where I don't add to that, but I also don't have to audit to code for the dozens of places configuration is being loaded.

It's going to happen, I just am not sure if it'll happen after I re-write the backends or before.